libmscore/score.cpp

```diff
@@ -263,7 +263,7 @@ Score::Score()
       accInfo = "No selection";
       }
 
-Score::Score(MasterScore* parent)
+Score::Score(MasterScore* parent, bool forcePartStyle /* = true */)
```

Comment:
Isn't the meaning of the parameter "resetPartStyleToDefault"?

Reply:
Well the comment accompanying the code previously referred to 'forcing some settings that made sense for parts', so this parameter means to force changing some settings to values that would make sense for parts.

We don't want to do this when reading though, since it isn't written if it's false, but this code automatically sets it to true. This leads values being read as false to be set to true.

Reply:
I didn't see this, or it didn't register at the time, but it appears this is the cause of https://musescore.org/en/node/285781.  What I believe *should* happen is this:

On part *creation*, we inherit style settings from the score as a starting point, then we should apply "style for part" MSS file from preferences if set.  But on part *read*, we shoudn't do either of those things.  The style settings are only written if they differ from the global MuseScore defaults in "baseStyle" (the ones hard-coded in style.cpp).  Thus, we need to start from baseStyle when reading the part.